.github/workflows/ci_39.yml

```
@@ -19,7 +19,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: ['7.3', '7.4', '8.0']
+        php: ['7.3', '7.4']
         moodle-branch: ['MOODLE_39_STABLE']
         database: [pgsql]
 
```

```suggestion
include:
  - php: '8.0'
    moodle-branch: 'MOODLE_311_STABLE'
    database: pgsql
```
Could this be done like this? Referencing idea from https://ncorti.com/blog/howto-github-actions-build-matrix in part `Shadow CI Jobs` and therefore external 311 ci-file becomes unneeded?

---

Did this. Also got rid of the coverage visualization for a while - there is some issues with it. Unit tests run only with tests for Moodle 3.9.